src/pages/analytics/analytics.scss

```diff
@@ -0,0 +1,7 @@
+page-analytics {
+    font-family: 'Open Sans', sans-serif;
```

Comment:
we should be able to inherit the main font from the body no need to explicitly declare them there. I know we haven't set a standard yet but it might be a good place to start.